### PR TITLE
#160845499 Users should be able to author, read, edit and delete articles

### PR DESCRIPTION
**Pull request by reiosantos**
#### What does this PR do?
- Enables a user to create, read, edit and delete articles

#### Description of Task to be completed?
- Add CRUD functionality,

#### How should this be manually tested?

- The functionality can be reviewed from [this address](https://ah-frontend-staging-pr-33.herokuapp.com)

#### What are the relevant pivotal tracker stories?
Articles: [#160845499](https://www.pivotaltracker.com/story/show/160845499)

Tags: [#160845506](https://www.pivotaltracker.com/story/show/160845506)

### CODE REVIEW

File: src/components/logo/Logo.scss [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+			-webkit-text-stroke: 15px darken(color(primary), 10%);
+		}
+		&:last-child:after {
+			-webkit-text-stroke: 17px hsla(0, 0%, 0%, .1);
+		}
+
+	}
+}
+
+//
```
Comment (Estaer) [inline src/components/logo/Logo.scss]: Consider removing all the commented out code in case you don't need it.

Comment (huxaiphaer) [reply src/components/logo/Logo.scss]: @Estaer , we are still resolving the issues.

Comment (Estaer) [reply src/components/logo/Logo.scss]: Cool

File: package.json [hunk 2]
```diff
@@ -4,20 +4,27 @@
...
     "file-loader": "^2.0.0",
+    "font-awesome": "^4.7.0",
     "history": "latest",
     "jimp": "^0.5.4",
     "jquery": "^3.3.1",
     "jwt-decode": "^2.2.0",
+    "moment": "^2.22.2",
     "prop-types": "latest",
     "react": "^16.5.2",
+    "react-chips": "^0.8.0",
```
Comment (isheebo) [inline package.json]: I am not sure, but if this is used for tags, kindly remove it

Comment (huxaiphaer) [reply package.json]: Ok @isheebo

**Comment by DrKimpatrick** [comment]
@huxaiphaer and @reiosantos thanks for the great work. You have two commit messages on this PR yet we need one. You need to resolve it.

**Comment by huxaiphaer** [comment]
@DrKimpatrick , We are going to resolve that.

File: src/containers/createArticle.js [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+	constructor(props) {
+		super(props);
+
+		this.state = {
+			title: '', titleError: '', description: '',
+			descriptionError: '', body: '', bodyError: ''
+		};
+
+		this.handleSubmit = this.handleSubmit.bind(this);
+		this.handleChange = this.handleChange.bind(this);
```
Comment (collinewait) [inline src/containers/createArticle.js]: `handleChange`  is an arrow function so binding it in the constructor is unnecessary.

**Comment by reiosantos** [comment]
> @huxaiphaer and @reiosantos thanks for the great work. You have two commit messages on this PR yet we need one. You need to resolve it.

This Issue @DrKimpatrick has been resolved, you can check it out to confirm.

File: src/containers/createArticle.js (hunk 3 above)
Comment (reiosantos) [reply src/containers/createArticle.js]: This has been resolved @collinewait, feel free to check it out.

File: src/static/js/material-dialog.js [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+/*
+ * Instance of application templates
+ */
+MaterialDialog.templates = {
+	alert:
+		"<div class='modal material-alert'>" +
+			"<div class='modal-content'><h4 class='modal-title'></h4></div>" +
+			"<div class='modal-footer'><div>" +
+		'</div>',
+	/* modal clases:
```
Comment (collinewait) [inline src/static/js/material-dialog.js]: Commented code.

**Pull request merged**